swift/apple/FirezoneKit/Sources/FirezoneKit/Features/SettingsView.swift

```diff
@@ -154,6 +188,97 @@ public struct SettingsView: View {
     model.load()
     dismiss()
   }
+
+#if os(macOS)
+  func exportLogsButtonTapped() {
+    self.isExportingLogs = true
+
+    let savePanel = NSSavePanel()
+    savePanel.prompt = "Save"
+    savePanel.nameFieldLabel = "Save log zip bundle to:"
+    savePanel.nameFieldStringValue = "firezone-logs.zip"
+
+    guard let window = NSApp.windows.first(where: { $0.identifier?.rawValue.hasPrefix("firezone-settings") ?? false }) else {
+      self.isExportingLogs = false
+      logger.log("Settings window not found. Can't show save panel.")
```

Comment:
What log level does this correspond to in `Console.app`? Would it make sense to use an explicit level here or is this standard practice?

Reply:
`log()` is like `debug()` but a little better. `log()` writes the the log to disk; `debug()` keeps it in memory only.

Actually, it's better we change all `debug()`s to `log()`s. I'll do that.

Reply:
Changed to `log()`s in commit 1e858e6.